### ISSUE TITLE
District of Columbia returning NONE for Zip Code

### ISSUE THREAD

**Issue by JeremyIR** [synthetichealth/synthea#1002]
Generating synthea data for BFD using the national sh script does not work for District of Columbia. It should be fixed by capitalizing it to "District Of Columbia" so that CMSStateCodeMapper.getStateCode does not treat DC zipcodes as NONE.

**Comment by hadleynet** [comment]
Good catch. We use a mix of "of" and "Of" in the codebase, we really need to settle on one and change it everywhere to match.

**Comment by jawalonoski** [comment]
Fixed with #993
